README.md

```diff
@@ -162,225 +189,414 @@ and modify the `system-reserved` line under `kubeletExtraArgs`:
 
 ### Option 1: Onboard Application with the Konflux UI
 
 With this approach, Konflux can create:
-1. The manifests in GitHub for the pipelines it will run against the applications
-   onboarded to Konflux.
-2. The Quay.io repositories into which it will push container images.
-
-The former is enabled by creating the
-[GitHub Application Secrets](./docs/github-secrets.md) **on all 3 namespaces** and
-installing your newly-created GitHub app on your repository, as explained above.
-
-To achieve the latter follow the step below:
-
-:gear: Create an organization and an application in Quay.io that will allow Konflux to
-create repositories for your applications. To do that,
-[Follow the procedure](./docs/quay.md#automatically-provision-quay-repositories-for-container-images)
-to configure a Quay.io application and deploy `image-controller`.
-
-#### Create Application and Component via the Konflux UI
-
-:gear: Follow these steps to onboard your application:
-
-1. Login to [Konflux](https://localhost:9443) as `user2@konflux.dev` (password:
-   `password`).
-2. Click `Create application`
-3. Verify the workspace is set to `user-ns2` (notice the `ws` breadcrumb trail just
-   above `Create an application` and click the `...` to switch workspaces as needed).
-4. Provide a name to the application and click "Add a component"
-5. Under `Git repository url`, copy the **https** link to your fork. This should
-   be something similar to `https://github.com/<your-name>/testrepo.git`.
-6. Leave `Docker file` blank. The default value of `Dockerfile` will be used.
-7. Under the Pipeline drop-down list, select `docker-build-oci-ta`.
-8. Click `Create application`.
-
-**NOTE:** If you encounter `404 Not Found` error, refer to the
-[troubleshooting guide](./docs/troubleshooting.md#unable-to-create-application-with-component-using-the-konflux-ui).
-
-The UI should now display the Lifecycle diagram for your application. In the Components
-tab you should be able to see your component listed and you'll be prompted to merge the
-automatically-created Pull Request (don't do that just yet. we'll have it merged in
-section [Trigger the Release](#trigger-the-release)).
-
-**NOTE:** if you have NOT completed the Quay.io setup steps in the previous section,
-Konflux will be UNABLE to send a PR to your repository. Konflux will display "Sending
-Pull Request".
-
-In your GitHub repository you should now see a PR was created with two new pipelines.
-One is triggered by PR events (e.g. when PRs are created or changed), and the other is
-triggered by push events (e.g. when PRs are merged).
-
-Your application is now onboarded, and you can continue to the
+
+1. Manifests in GitHub for the application pipelines it will run.
+
+2. The [Quay.io](https://quay.io) repositories into which it will push container images.
+
+You can configure the first item by creating
+[GitHub Application Secrets](https://github.com/konflux-ci/konflux-ci/blob/main/docs/github-secrets.md)
```

Comment:
Is there a benefit of having the complete URL here comparing to having only the path inside the repository? I think that having the full path is reducing the portability of this document (i.e. if the entire repo is moved).